src/main/java/wayoftime/bloodmagic/client/hud/element/ElementDivinedInformation.java

```diff
@@ -25,39 +26,20 @@ public ElementDivinedInformation(int lines, boolean simple, Class<T> tileClass)
 	public boolean shouldRender(Minecraft minecraft)
 	{
 		Player player = Minecraft.getInstance().player;
-		ItemStack sigilStack = player.getItemInHand(InteractionHand.MAIN_HAND);
-		boolean flag = false;
-		if (simple)
-		{
-			if (sigilStack.getItem() == BloodMagicItems.DIVINATION_SIGIL.get() || sigilStack.getItem() == BloodMagicItems.SEER_SIGIL.get())
-				flag = true;
-			else
-				flag = isFlagSigilHolding(sigilStack, true);
 
-			if (!flag)
-			{
-				sigilStack = player.getItemInHand(InteractionHand.OFF_HAND);
-				if (sigilStack.getItem() == BloodMagicItems.DIVINATION_SIGIL.get() || sigilStack.getItem() == BloodMagicItems.SEER_SIGIL.get())
-					flag = true;
-				else
-					flag = isFlagSigilHolding(sigilStack, true);
-			}
+		NonNullList<ItemStack> inventory = InventoryHelper.getActiveInventories(player);
 
-		} else
+		boolean flag = false;
+		for (int i = 0; i < inventory.size(); i++)
 		{
-			if (sigilStack.getItem() == BloodMagicItems.SEER_SIGIL.get())
+			ItemStack sigilStack = inventory.get(i);
```

Comment:
Optional: This For loop and `inventory.get(i)` could be replaced with an Enhanced For loop.  
`for (ItemStack sigilStack : inventory)`

I have also noticed that the Seer's Sigil in a Sigil of Holding does not work with the Incense Altar.  I've replicated the problem on the main branch so it isn't caused by this PR, but might be worth looking into.

Reply:
It looks like the Seers Sigil in a Sigil of Holding issue for the Incense Altar is fixed by removing the `&& !simple` check for the Seer's Sigil on line 56.

The Seer's Sigil only has a "simple" overlay, so that check is what is throwing it out when in the Sigil of Holding, where as it doesn't get thrown out when it is out of the Sigil of Holding.

This does show another known issue where both the Simple and non-Simple overlays get rendered at the same time with the Seer's Sigil, but that is only a problem on the Blood Altar and only when the player has moved the overlay themselves (they default to being perfectly aligned on top of each other so is not a problem).  Changing that should likely be a separate PR.

Reply:
Adjusted the for loop and removed the mentioned `&& !simple` check

Is there a reason not to just remove the simple part from the non-simple overlay? Would probably also need to adjust the default position its rendered at

Reply:
That is how I would start trying to address that issue, but IMO that starts to exceed the scope of an expanded Curios Compat PR.  When I first started using GitHub I was told that it is best to try to keep each Pull Request focused on one purpose.

I only brought up the Sigil of Holding + Incense Altar issue because I was pretty certain the fix would be a small error in the file you were already editing, and it directly affected the Seer's Sigil usability as a Curio.

If you want to tackle the double-overlay issue then I would suggest making it a separate PR (it shouldn't affect curios compatibility).  You might want to move the `&& !simple` check to that one if you do make it.

I think this PR is good to go as it is now, though I'm leave it up to WayofTime (or TehNut [Nick]) for final approval/Pull.  Way has been pretty busy so I don't know how long that will take.